compiler/optimizer/InductionVariable.cpp

```diff
@@ -7231,21 +7232,25 @@ TR_InductionVariableAnalysis::getEntryValue(TR::Block *block,
 
    // Did not find the store in the block
    //
-   TR_PredecessorIterator pit(block);
-   TR::Node *defValue = (TR::Node *)-1;
-   for (TR::CFGEdge *edge = pit.getFirst(); edge; edge = pit.getNext())
+   const int32_t maxDepth=500;
```

Comment:
Can you make this settable by a environment variable, defaulting to 500?

Reply:
How does "TR_IVAMaxDepth" sound? Maybe you or @jdmpapin can think of a more appropriate env var name?

Reply:
`TR_IVAEntryValueMaxDistance`?

Reply:
`TR_IVAEntryValueMaxDistance` seems reasonable to me.

Reply:
> `TR_IVAEntryValueMaxDistance` seems reasonable to me.

@0xdaryl  Oops.. I just saw your comment just after pushing a change to use TR_IVAEntryValueMaxDepth. I'll change it if you think it's worth doing??

Reply:
No problem.  I'm fine with what you have.